controls/V-71929.rb

```diff
@@ -42,8 +42,15 @@
   tag cci: ["CCI-000199"]
   tag nist: ["IA-5 (1) (d)", "Rev_4"]
 
-  describe login_defs do
-    its('PASS_MAX_DAYS.to_i') { should cmp <= 60 }
+  unless command("grep -ie '^[^#]*NOPASSWD' /etc/sudoers /etc/sudoers.d/*").stdout.empty?
```

Comment:
I am not sure I agree with this approach - it seems to only touch the sudo system - how does this tell us that passwords are not enabled on the system? I would think this happens more in the pam_ system or sssd system.